src/app/archive/components/archive/archive.component.ts

```diff
@@ -1,128 +1,64 @@
 
 @Component({
   selector: 'app-archive',
   templateUrl: './archive.component.html',
-  styleUrls: ['./archive.component.scss'],
-  animations: [
-    trigger('detailExpand', [
-      state('collapsed', style({ height: '0px', minHeight: '0' })),
-      state('expanded', style({ height: '*' })),
-      transition(
-        'expanded <=> collapsed',
-        animate('225ms cubic-bezier(0.4, 0.0, 0.2, 1)')
-      ),
-    ]),
-  ],
+  styleUrls: ['./archive.component.scss']
 })
-export class ArchiveComponent implements OnInit {
+export class ArchiveComponent {
   @Select(ArchiveState.filteredItems) items$: Observable<Item[]>;
-  @Select(ArchiveState.filter) filter$: Observable<Filter>;
-  @Select(ArchiveState.itemById) itemById$: Observable<Item>;
-
-  public filter: Filter;
-  public items: Item[];
-  public displayedColumns = [
-    'content',
-    'open_timestamp',
-    'close_timestamp',
-    'result_score',
-  ];
+  @Select(ArchiveState.filter) filter$: Observable<CaseFilter>;
+  @Select(ArchiveState.sort) sort$: Observable<CaseSort>;
 
   public archiveQuestions: any[] = [
-      {
-        title: 'Wann sehe ich etwas im Archiv?',
-        description: 'Hier kommt einiges zusammen. Genaue Details findest du hier.',
-        background: 'color__bittersweet',
-        icon: 'fal fa-eye'
-      },
-      {
-        title: 'Wie errechnet sich der Score?',
-        description: 'Eine Anleitung für genau solche Fälle findest du auf dieser Seite.',
-        background: 'color__neon-blue',
-        icon: 'fal fa-chart-bar'
-      },
-      {
-        title: 'Eine weitere Frage?',
-        description: 'Und hier ein weiterer Beschreibungstext, der erklärt, was mich beim Klick darauf erwartet.',
-        background: 'color__purple',
-        icon: 'fal fa-leaf'
-      },
-      {
-        title: 'Eine weitere Frage?',
-        description: 'Und hier ein weiterer Beschreibungstext, der erklärt, was mich beim Klick darauf erwartet.',
-        background: 'color__bittersweet',
-        icon: 'fal fa-leaf'
-      },
-    ];
+    {
+      title: 'Wann sehe ich etwas im Archiv?',
+      description: 'Hier kommt einiges zusammen. Genaue Details findest du hier.',
+      background: 'color__bittersweet',
+      icon: 'fal fa-eye'
+    },
+    {
+      title: 'Wie errechnet sich der Score?',
+      description: 'Eine Anleitung für genau solche Fälle findest du auf dieser Seite.',
+      background: 'color__neon-blue',
+      icon: 'fal fa-chart-bar'
+    },
+    {
+      title: 'Eine weitere Frage?',
+      description: 'Und hier ein weiterer Beschreibungstext, der erklärt, was mich beim Klick darauf erwartet.',
+      background: 'color__purple',
+      icon: 'fal fa-leaf'
+    },
+    {
+      title: 'Eine weitere Frage?',
+      description: 'Und hier ein weiterer Beschreibungstext, der erklärt, was mich beim Klick darauf erwartet.',
+      background: 'color__bittersweet',
+      icon: 'fal fa-leaf'
+    }
```

Comment:
What do you think of putting static data like this in an assets JSON file?
To me, it would make the component code itself more readable.

Reply:
Either as static data or it could also be fetched from the backend(?). These are only placeholders yet, I think.